src/events/nats.js

```diff
@@ -0,0 +1,170 @@
+/* eslint-disable camelcase */
```

Comment:
Disable it only where needed

Reply:
If I remove this line, I need to add it again 27 times in the entire file.
I want to keep the same coding style (not camel case) of nats variables for consistency.